scripts/pi-hole/js/queries.js

```diff
@@ -388,33 +389,83 @@ $(function () {
       var api = this.api();
 
       // Query type IPv4 / IPv6
-      api.$("td:eq(1)").click(function (event) {
-        addColumnFilter(event, 1, this.textContent);
-      });
+      api
+        .$("td:eq(1)")
+        .click(function (event) {
+          addColumnFilter(event, 1, this.textContent);
+        })
+        .hover(
+          function () {
+            $(this).css("cursor", "pointer").attr("title", tooltipText(1, this.textContent));
+          },
+          function () {
+            $(this).css("cursor", "auto");
```

Comment:
Please don't do this. We already have CSS classes to use. It's always more performant to add/remove CSS classes that this.

Also, do note that this does not fix anything accessibility related. It's just a visual indication only.

Reply:
So what is the right way to do this?

Reply:
`addClass('pointer')`/`removeClass('pointer')`

There is a reason we don't have such utils upstream BTW. Because adding a cursor does not improve accessibility, it's only a visual indication, which on non-pointer devices does not help at all. :)

Reply:
Thanks. I don't think the web interface is meant or designed for non-pointer devices. I will check, nonetheless, if we can have the same effect with a stepping through using <kbd>Tab</kbd> and then maybe <kbd>Ctrl</kbd>+<kbd>Space</kbd>.